helpers/cleanup.sh

```diff
@@ -14,12 +14,25 @@
 # limitations under the License.
 
 
-PROJECT_ID="$1"
+PROJECT_ID="$(gcloud projects list --format="value(projectId)" | grep -Eo "$1")"
```

Comment:
If I'm reading this correctly, we default to using the gcloud project ID? Instead I think any passed project ID should take precedence.

Reply:
This line basically checks to see if the project exists. First, we get a list of projects for the authenticated user, and then there's a check to see if the project the user has supplied exists. If it doesn't, then an error gets thrown. So the project id that is being used is whatever the user is providing.

Reply:
Actually, this is cleaner:

PROJECT_ID="$(gcloud projects list --format="value(projectId)" --filter=$1)"